src/popup/utils/connection.js

```diff
@@ -47,11 +47,7 @@ export const ensureContentScriptInitialized = async () => {
     });
     contentPostMessage = async (message, tab) => {
       const id = genUuid();
-      try {
-        await browser.tabs.sendMessage(tab, { data: { ...message, uuid: id } });
-      } catch (e) {
-        return Promise.resolve(e);
```

Comment:
Well, I did not noticed `resolve` instead of `reject`. @petbaik Is it intentional to return an exception as a result? If the answer is yes then let's close this PR, but I think there should be a more obvious solution.

Reply:
Actually it's not necessary to return exception just promise need to be resolved in case of exception so code can continue without errors https://github.com/aeternity/superhero-wallet/blob/develop/src/store/actions.js#L265

Reply:
Can we catch this error there instead?

Reply:
I think we can

Reply:
Well, I did that, can we merge it now?